htdocs/class/xoopsform/formcolorpicker.php

```diff
@@ -54,8 +54,11 @@ public function render()
             echo '<script type="text/javascript" src="' . XOOPS_URL . '/include/color-picker.js"></script>';
         }
         $this->setExtra(' style="background-color:' . $this->getValue() . ';"');
-
-        return parent::render() . "<input type='reset' value=' ... ' onclick=\"return TCP.popup('" . XOOPS_URL . "/include/',document.getElementById('" . $this->getName() . "'));\">";
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            return substr(parent::render(), 0,-6) . "<input class='form-control' type='reset' value=' ... ' onclick=\"return TCP.popup('" . XOOPS_URL . "/include/',document.getElementById('" . $this->getName() . "'));\"></div>";
+        } else {
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->